# This message passes an integer which maps to a state of the robot.
uint16 ESTOP=0
uint16 TELEOP=1
uint16 WALL_FOLLOW=2
uint16 MAINTAIN_ANGLE=3
uint16 MAINTAIN_DISTANCE=4
uint16 FOLLOW=5
uint16 state
